code/src/styles/App.scss

```diff
@@ -0,0 +1,36 @@
+@import '../variables.scss';
+
+.App {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: #f7d4b8;
+  height: 100vh;
+  font-family: 'Montserrat', sans-serif;
+  color: rgb(247, 212, 184);
+
+  &__main {
```

Comment:
You seem to impress me with every next project Sara. Great usage of ampersand, this is exactly how we should combine BEM with Sass features!